model_analyzer/config/generate/brute_run_config_generator.py

```diff
@@ -70,8 +69,12 @@ def get_configs(self):
 
     def _get_next_config(self):
         yield from self._generate_subset(0, default_only=True)
-        self._default_returned = True
-        yield from self._generate_subset(0, default_only=False)
+        if self._do_search():
```

Comment:
this function name might be misleading.  search_disable=true for us means that we still do manual mode (and do_search would still return true in that case)

I could say call it _should_generate_non_default_only_configs(), but that seems long? Thoughts?

Reply:
How about `_only_generate_non_default_configs()`?